tests/jax/components/executing/action_selection_test.py

```diff
@@ -41,13 +42,13 @@ def dummy_config() -> ExecutorSelectActionConfig:
 
 
 @pytest.fixture
-def dummy_empty_config() -> ExecutorSelectActionConfig:
-    """Dummy empty config attribute for FeedforwardExecutorSelectAction class
+def dummy_ff_executor_select_action():
```

Comment:
I would rather call this `test_ff_selection_action`, or just `ff_selection_action`, since you're not constructing a dummy version of it like you are for the executor. You're using the actual component, as you should.